### ISSUE TITLE
ftp速度非常慢，并且rz -by命令上传文件的时候导致其它窗口相应迟缓

### ISSUE THREAD

**Issue by jaychoubaby** [kingToolbox/WindTerm#1701]
工具的ftp并不好用，因为速度非常慢；

于是我使用rz -by命令上传，但是我发现它的速度对比其它工具（secureCRT）来说也慢很多，并且在某一个窗口上传中的时候会影响到其它窗口的使用体验，希望有时间可以进行优化。

**Comment by kingToolbox** [comment]
There seem to be bugs in the SFTP transmission. Both #437 and #687 have mentioned this issue, and the transfer speed was fast before the `WindTerm 2.3.0` version. It seems that a feature introduced later may have caused this problem. I will examine the changes made in the `2.3.0` version and try to identify the root cause of the issue.

I will investigate the speed issue of `rz -by` as well. If other windows are connected to the same server as the current transfer window, it's normal that they may respond slowly, but if they are not, I may need to optimize the program. However, it's common for reading files from local disks to affect program responsiveness, as it involves significant overhead.

**Comment by jaychoubaby** [comment]
> There seem to be bugs in the SFTP transmission. Both #437 and #687 have mentioned this issue, and the transfer speed was fast before the `WindTerm 2.3.0` version. It seems that a feature introduced later may have caused this problem. I will examine the changes made in the `2.3.0` version and try to identify the root cause of the issue.
> 
> I will investigate the speed issue of `rz -by` as well. If other windows are connected to the same server as the current transfer window, it's normal that they may respond slowly, but if they are not, I may need to optimize the program. However, it's common for reading files from local disks to affect program responsiveness, as it involves significant overhead.

You are right. It may indeed be a performance issue caused by uploading on the same server connection. I just tested it and it wasn't as severe as yesterday. Thank you for your response and help.
